pom-explorer-core/src/main/java/fr/lteconsulting/pomexplorer/model/Dependency.java

```diff
@@ -10,6 +11,7 @@
 	private final String artifactId;
 	private final String version;
 	private final Scope scope;
+	private final Boolean isVersionSelfManaged;
```

Comment:
I am not very hot with this change. `Dependency` IMHO should stay close to the Maven project structure (it was conceived like that, like the other classes : DependencyKey, ...)
Cannot we just create a AnaylzerHelper class which would contain the new 'getVersionScopeFromDependencyManagement' ?

Reply:
We could introduce a helper. However, with this version we have fast access, otherwise we need to re-analyse the MavenProject over and over again.